tools/src/test/java/com/nedap/archie/json/RMJacksonTest.java

```diff
@@ -22,7 +22,7 @@
     @Test
     public void parseEhrBaseJsonExample() throws Exception {
         try(InputStream stream = getClass().getResourceAsStream("pablos_example.json")) {
-            RMJacksonConfiguration configuration = new RMJacksonConfiguration();
+            ArchieJacksonConfiguration configuration = ArchieJacksonConfiguration.createStandardsCompliant();
             configuration.setTypePropertyName("_type");
```

Comment:
```suggestion
            ArchieJacksonConfiguration configuration = ArchieJacksonConfiguration.createStandardsCompliant();
```
The `setTypePropertyName` is no longer needed.

Reply:
good catch.